packages/compass-indexes/src/components/create-index-modal/create-index-modal.jsx

```diff
@@ -329,54 +348,71 @@ class CreateIndexModal extends PureComponent {
           <form
             name="create-index-modal-form"
             onSubmit={this.onFormSubmit.bind(this)}
-            data-test-id="create-index-modal">
-
+            data-test-id="create-index-modal"
+          >
             <ModalInput
-              autoFocus
               id="create-index-name"
               name="Choose an index name"
               value={this.props.name}
-              onChangeHandler={(evt) => (this.props.changeName(evt.target.value))} />
+              onChangeHandler={(evt) => this.props.changeName(evt.target.value)}
+            />
 
             <div className={styles['create-index-modal-fields']}>
-              <p className={styles['create-index-modal-fields-description']}>Configure the index definition</p>
+              <p className={styles['create-index-modal-fields-description']}>
+                Configure the index definition
+              </p>
               {this.renderIndexFields()}
 
               <button
                 onClick={this.handleAddField.bind(this)}
                 id="add-field"
-                className="btn btn-primary btn-sm btn-full-width">
+                className="btn btn-primary btn-sm btn-full-width"
+              >
                 Add another field
               </button>
             </div>
 
-            <div className={styles['create-index-modal-toggle-bar']}
-              onClick={this.handleToggleBarClick.bind(this)}>
+            <button
+              className={styles['create-index-modal-toggle-bar']}
+              onClick={this.handleToggleBarClick.bind(this)}
```

Comment:
We're changing this to a button as it was a `div` with an `onClick` before. Happy that lint picks this up now.
> Visible, non-interactive elements with click handlers must have at least one keyboard listener.eslint [jsx-a11y/click-events-have-key-events](https://github.com/evcohen/eslint-plugin-jsx-a11y/tree/master/docs/rules/click-events-have-key-events.md)

> Static HTML elements with event handlers require a role.eslint [jsx-a11y/no-static-element-interactions](https://github.com/evcohen/eslint-plugin-jsx-a11y/tree/master/docs/rules/no-static-element-interactions.md)

Reply:
:+1:

Do we want to make this an `import(compass-components).Accordion` at some point?

Reply:
Good suggestion, I think so yes, we have an upcoming epic for updating the indexes page to leafy green, so I was thinking to do it then:
https://jira.mongodb.org/browse/COMPASS-5485
I started to update to the Accordion it in this pr, but because a lot of these ui states are tied into the store it ended up changing a bit too much logic and I wanted to avoid too many changes for this pr.